test/models/ldap_config_test.rb

```diff
@@ -231,7 +231,13 @@ class LdapConfigTest < ActiveSupport::TestCase
   test 'massive import' do
     user         = users(:supervisor)
     organization = organizations(:google)
-    emails_count = NOTIFY_NEW_ADMIN ? 2 : 1
+    emails_count = if SHOW_WEAKNESS_EXTRA_ATTRIBUTES
+                    0
+                  elsif NOTIFY_NEW_ADMIN
+                    2
+                  else
+                    1
+                  end
```

Comment:
Le falta un espacio a cada línea de estas:
```suggestion
                     0
                   elsif NOTIFY_NEW_ADMIN
                     2
                   else
                     1
                   end
```